packages/iov-multichain/src/jsonrpcsigningserver.spec.ts

```diff
@@ -247,6 +247,7 @@ describe("JsonRpcSigningServer", () => {
       method: "signAndPost",
       params: {
         reason: "string:Please sign",
+        identity: TransactionEncoder.toJson(signer),
```

Comment:
Can we call this `signer` and update https://github.com/iov-one/iov-core/blob/master/docs/out-of-process-rpc.md?

Reply:
Updated. It's a little confusing that we also use `signer` in lots of examples to refer to an instance of `MultiChainSigner` though.

Reply:
Hmm, yes, right, good point. The tool and the tool's user have the same name. I don't see an easy way to resolve this right now.

Reply:
Red signer and blue signer?

Reply:
"Open signer" would be good for a tool that is open to be used by anybody, such as closed signers.